### DISCUSSION TITLE
Dynamic Linking/Representation from one Drawing to Another?

### DISCUSSION THREAD

**Discussion by sl3uth** [jgraph/drawio#2291] [Q&A]
Is it possible to include a Drawing selection or tab into another Drawing, so it is linked rather than copied and distinct?

Example:
Tab 1 - Overview of complex process containing shrunk versions of the highly detailed drawings on subsequent Tabs
Tab 2 - highly detailed drawing
...
Tab n - highly detailed drawing

If I edit the detailed drawing of Tab 2, the linked (and scaled down) version of the drawing on Tab 1 contains the changes because it is dynamically linked to the Tab 2 drawing (at least one-way, where the representation of the Tab 2 Drawing on Tab 1 would require going to Tab 2 to edit).

Thank you!

**Comment by davidjgraph** [comment]
Do it as a background page, maybe?

**Reply by sl3uth** [reply]
I'm not sure I understand, but I'll try to take a look at using a background page.

_Since I'm not sure I'm communicating my request clearly, here's a bit more description:_
Another aspect is the ability to print out the overview of Tab 1 (from my original example), showing how all the sub-processes link together, and yet also be able to easily print out each detailed sub-process discretely.

In CAD programs, they accomplish some of these things by distinguishing between the drawing(s) and the view(s), where the views allow the ability to bring multiple drawings (or portions thereof) into a logical arrangement to communicate the relationships.  Then all editing in the drawing(s) is simply reflected in the view(s) without copy/pasting and running into synchronization issues between detailed drawings and their representations when combined with other drawing information in a view.
